schedule/tests.py

```diff
@@ -49,3 +51,36 @@ def test_GET_from_scheduleId_company_2(self):
         self.assertIsNotNone(response)
         self.assertEquals(status.HTTP_200_OK, response.status_code)
         self.assertContains(response, "GENERATED SCHEDULE FROM COMPANY 2 ALGORITHM")
+
+# ADAPTER TESTS
+    def test_none(self):
+        course_dict = course_to_dictionary(None)
+        self.assertIsNone(course_dict)
+
+    def test_trivial(self):
+        course_attributes = {
```

Comment:
Just as a heads up, the `Course` object for the Algorithms team does not align with the one for the FE team - this object should reflect the algs team spec